.github/workflows/reusable_provenance.yaml

```diff
@@ -128,50 +128,35 @@ jobs:
         run: ls -R
         working-directory: downloads
 
-      - name: Upload binary to Ent
+      - name: Upload binary, provenance, signed tag to Ent
         id: ent_upload_binary
         working-directory: downloads
```

Comment:
merged into a single step, as otherwise we'd have to pass a variable amount of variables with variable names between steps. It becomes unmaintainable quickly.